packages/pn-commons/src/components/NotificationDetail/NotificationPaymentF24Item.tsx

```diff
@@ -179,14 +178,15 @@ const NotificationPaymentF24Item: React.FC<Props> = ({
         gap={0.5}
         flexDirection="column"
         flex="1 0 0"
+        mb={isMobile ? '8px' : 0}
```

Comment:
use spacing on upper Stack as suggested in the previous review

Reply:
done